dbld/builddeps

```diff
@@ -92,9 +125,9 @@ function install_gradle {
     mkdir -p /opt/gradle
     unzip -d /opt/gradle /tmp/gradle.zip
     rm -rf /tmp/gradle.zip
+    rm -rf /usr/bin/gradle
     ln -s /opt/gradle/gradle-${GRADLE_VERSION}/bin/gradle /usr/bin/gradle
-    find / -name 'libjvm.so' | sed 's@/libjvm.so@@g' | tee --append /etc/ld.so.conf.d/openjdk-libjvm.conf
-    ldconfig
+    set_jvm_paths $@
```

Comment:
I don't see any image that would use the argument to set_jvm_paths, could that be removed?